src/components/shared/PartnershipsDropDown.tsx

```diff
@@ -7,24 +7,23 @@ import { withLocaleUrl } from "../../utils/utils";
 
 export default function PartnershipsDropDown() {
```

Comment:
Is this component being used anymore. Looks like a dupe of some of the code in Header.tsx

Reply:
removed!